x/gov/codec.proto

```diff
@@ -40,10 +38,12 @@ message ElectionRule {
   uint32 version = 2;
   // Admin is the address that is allowed ot modify an existing election rule.
   bytes admin = 3 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
+  // ElectorateID references the electorate using this rule (without version, as changing electorate changes the rule)
```

Comment:
The Rule references the electorate in general.
When a Proposal is created it references one version.

This is essential to allow an update of the electorate to update the election rule as well... Otherwise, we change the electorate, but all votes are still pinned to an older version.